### ISSUE TITLE
 support ubuntu 19.04

### ISSUE THREAD

**Issue by 29e7e280-0d1c-4bba-98fe-f7cd3ca7500a** [JonasGroeger/jetbrains-ppa#27]
need  support ubuntu 19.04

**Comment by JonasGroeger** [comment]
Codename: `disco`

As a workaround you can edit this file: `/etc/apt/sources.list.d/jetbrains-ppa.list`.

Replace whatever comes before `main` with `cosmic`. Example:

```
deb http://jetbrains-ppa.s3-website.eu-central-1.amazonaws.com disco main
deb http://jetbrains-ppa.s3-website.eu-central-1.amazonaws.com cosmic main

**Comment by gsbhasin123** [comment]
Yeah, I want the same for 19.10 @JonasGroeger

**Comment by sor** [comment]
It does just work fine with the "wrong name", I used this PPA for Rider with every Version from 18.04 to 19.10.

**Comment by gsbhasin123** [comment]
Ah okay, nice :P

**Comment by JonasGroeger** [comment]
fixed with #37
